README.md

```diff
@@ -451,8 +451,7 @@ The artifact is also renamed to make its intention more obvious. Documentation f
 Advanced topics are available in [the official docs](https://www.apollographql.com/docs/android/):
 
 * [caching.md](https://www.apollographql.com/docs/android/essentials/caching/)  
-* [plugin-configuration.md](https://www.apollographql.com/docs/android/gradle/plugin-configuration/) 
```

Comment:
These links were invalid. Somehow gatsby was ok with them

Reply:
Thanks for the heads up.